### PR TITLE
Add fraxswap to `dex.trades_beta`

### PR DESCRIPTION
**Pull request by tomfutago**
linked to #4759

### CODE REVIEW

**Comment by Hosuke** [comment]
Since the hardest part in conflicts resolving are merging the testing part, I will try to submit a PR to simplify the tests format a bit.

File: models/_sector/dex/trades/ethereum/_schema.yml [hunk 1]
```diff
@@ -272,3 +271,23 @@ models:
...
+    config:
+      tags: ['ethereum', 'dex', 'trades', 'fraxswap', 'uniswap', 'v2']
+    description: "Fraxswap ethereum base trades"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
+      - check_dex_base_trades_seed:
+          seed_file: ref('apeswap_ethereum_base_trades_seed')
```
Comment (jeff-dude) [inline models/_sector/dex/trades/ethereum/_schema.yml:290]: looks like accidental left behind due to merge conflict fix?

side note, we've set ourselves up for resolving lots of merge conflicts post-merge on each PR 😵‍💫

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/_sector/dex/trades/ethereum/_schema.yml (hunk 1 above)
Comment (Hosuke) [inline models/_sector/dex/trades/ethereum/_schema.yml:290]:
```suggestion
          seed_file: ref('fraxswap_ethereum_base_trades_seed')            
```

File: models/_sector/dex/trades/ethereum/_schema.yml [hunk 2]
```diff
@@ -272,3 +271,23 @@ models:
...
+    config:
+      tags: ['ethereum', 'dex', 'trades', 'fraxswap', 'uniswap', 'v2']
+    description: "Fraxswap ethereum base trades"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
+      - check_dex_base_trades_seed:
+          seed_file: ref('fraxswap_ethereum_base_trades_seed')
```
Comment (Hosuke) [inline models/_sector/dex/trades/ethereum/_schema.yml:290]:
```suggestion
```

File: models/_sector/dex/trades/ethereum/_schema.yml (hunk 1 above)
Comment (tomfutago) [reply models/_sector/dex/trades/ethereum/_schema.yml]: Yes, looks like it.. Not sure if there is an easy way around it - usual joys of migration, same files are getting touched multiple times. 

One suggestion that might ease some pains at least a little bit is to update all `_schema`, `_source` and `dex_<project>_base_trades` to add commented out placeholders for all projects (even if it's just `# <project_name>`), which hopefully will minimise code blocks shifting around. Just a thought.

Comment (jeff-dude) [reply models/_sector/dex/trades/ethereum/_schema.yml]: if you and @Hosuke think that's worthwhile to do at this stage, plz do feel free 🤝 not a must either

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**